README.md

```diff
@@ -30,7 +30,7 @@ OpenCue provides the following features to help manage rendering jobs at scale:
 - You can split a host into a large number of procs, each with their own
```

Comment:
I think it might be more useful to move the link to this line, where the term is mentioned for the first time?

Reply:
Thanks, it's a good point. 
I will change it.